airbyte-integrations/connectors/source-iterable/source_iterable/api.py

```diff
@@ -54,13 +54,7 @@ def next_page_token(self, response: requests.Response) -> Optional[Mapping[str,
         """
         return None
 
-    def request_params(
-        self,
-        stream_state: Mapping[str, Any],
-        request_configuration: Optional[Mapping] = None,
-        next_page_token: Mapping[str, Any] = None,
-        parent_stream_record: Mapping = None,
-    ) -> MutableMapping[str, Any]:
+    def request_params(self, stream_state: Mapping[str, Any], **kwargs) -> MutableMapping[str, Any]:
```

Comment:
if we're not using any of the params maybe we can just do `**kwargs` for everything

Reply:
will update in #2895